pkg/mdformatter/mdgen/mdgen.go

```diff
@@ -84,7 +84,12 @@ func (t *genCodeBlockTransformer) TransformCodeBlock(ctx mdformatter.SourceConte
 			}
 			return nil, errors.Wrapf(err, "run %v, out: %v", execCmd, b.String())
 		}
-		return b.Bytes(), nil
+		output := b.Bytes()
+		// Add newline to output if not present.
+		if !bytes.HasSuffix(output, []byte("\n")) {
```

Comment:
```suggestion
		if !bytes.HasSuffix(output, newLineChar) {
```
.. to not creat slice every time? (micro-optimization, but also readability) and I think we already have newLineChar somewhere 🤔

Reply:
not a blocker

Reply:
Done!